diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/RunInScriptWarnTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.cqfn.diktat.ruleset.chapter6
+
+import org.cqfn.diktat.ruleset.constants.Warnings.RUN_IN_SCRIPT
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.chapter6.RunInScript
+import org.cqfn.diktat.util.LintTestBase
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class RunInScriptWarnTest : LintTestBase(::RunInScript) {
+    private val ruleId: String = "$DIKTAT_RULE_SET_ID:run-script"
+
+    @Test
+    @Tag(WarningNames.RUN_IN_SCRIPT)
+    fun `check simple example`() {
+        lintMethod(
+            """
+                class A {}
+                
+                fun foo() {
+                }
+                
+                diktat {}
+                
+                diktat({})
+
+                foo/*df*/()
+
+                foo( //dfdg
+                    10
+                )
+                println("hello")
+                
+                w.map { it -> it }
+                
+                tasks.register("a") {
```

Comment:
This should definitely be allowed, we'll break all gradle scripts otherwise. Lets allow any function calls on top level.